addons/sale_project/models/sale_order.py

```diff
@@ -151,7 +151,13 @@ def action_view_task(self):
 
         action = self.env["ir.actions.actions"]._for_xml_id("project.action_view_task")
         if self.tasks_count > 1:  # cross project kanban task
-            action['views'] = [[kanban_view_id, 'kanban'], [list_view_id, 'tree'], [form_view_id, 'form'], [False, 'graph'], [False, 'calendar'], [False, 'pivot']]
+            for idx, (view_id, view_type) in enumerate(action['views']):
+                if view_type == 'kanban':
+                    action['views'][idx] = (kanban_view_id, 'kanban')
+                elif view_type == 'tree':
+                    action['views'][idx] = (list_view_id, 'tree')
+                elif view_type == 'form':
+                    action['views'][idx] = (form_view_id, 'form')
```

Comment:
This code will no longer make sure the kanban and list views are the first views, right? 🧐 

If the views order defined in the action is already the one we expect then okay your fix is good.

Reply:
The order defined in the action is already the one we expect indeed.
Of course, if someone change the action (with studio), it will be reflected here, but I guess we don't really care

Reply:
I agree 🙂